enterprise/server/remote_execution/runner/BUILD

```diff
@@ -4,17 +4,17 @@ package(default_visibility = ["//enterprise:__subpackages__"])
 
 go_library(
     name = "runner",
-    srcs = ["runner.go"],
+    srcs = [
+        "runner.go",
+        "runner_linux.go",
+        "runner_nolinux.go",
```

Comment:
The `sandbox` isolation type is darwin-specific, so we probably want separate `_darwin` and `_windows` files.

Reply:
Did not know about this. Good call!